variables.tf

```diff
@@ -843,6 +855,30 @@ variable "wait_till" {
   }
 }
 
+variable "kube_version" {
+  description = "The version of the OpenShift cluster that should be provisioned (format 4.x). This is only used during initial cluster provisioning, but ignored for future updates. If no value is passed, or the string 'latest' is passed, the current latest OCP version will be used."
+  type        = string
+  default     = null
+  validation {
+    condition = anytrue([
+      var.kube_version == null,
+      var.kube_version == "latest",
+      var.kube_version == "4.8",
+      var.kube_version == "4.9",
+      var.kube_version == "4.10",
+      var.kube_version == "4.11",
+      var.kube_version == "4.12"
+    ])
+    error_message = "The specified kube_version is not one of the validated versions."
+  }
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
```

Comment:
This implies all resources will be tagged with these tags. But as far as I can see its only used by cluster?
So do we rename the variable here? Also what about resource tagging for other resources deployed?

Reply:
Duplicate variable, removing this.